app/components/FilterableTable/Filters/TableFilter.tsx

```diff
@@ -6,6 +6,7 @@ export interface ISearchOptionSettings {
   filterable?: boolean;
   sortable?: boolean;
   sortColumnName?: string;
+  renderEnumValue?: (x: string) => string;
```

Comment:
I'd still want to move this to the `EnumFilter` class instead, since this is specific to how enums are displayed

Reply:
ah, I left it in there to fix a typescript error on the settings object. I'll add a new interface inside EnumFilter and extend IsearchOptionSettings there with renderEnumValue